components/raftstore/src/store/peer.rs

```diff
@@ -1513,59 +1529,46 @@ where
             // We must renew current_time because this value may be created a long time ago.
             // If we do not renew it, this time may be smaller than propose_time of a command,
             // which was proposed in another thread while this thread receives its AppendEntriesResponse
-            //  and is ready to calculate its commit-log-duration.
+            // and is ready to calculate its commit-log-duration.
             ctx.current_time.replace(monotonic_raw_now());
         }
 
         if self.is_leader() {
             if let Some(hs) = ready.hs() {
-                // Correctness depends on the fact that the leader lease must be suspected before
-                // other followers know the `PrepareMerge` log is committed, i.e. sends msg to others.
-                // Because other followers may complete the merge process, if so, the source region's
-                // leader may get a stale data.
-                //
-                // Check the committed entries.
-                // TODO: It can change to not rely on the `committed_entries` must have the latest committed entry
-                // and become O(1) by maintaining these not-committed admin requests that changes epoch.
+                // Check the new committed entries.
```

Comment:
Should explain why check the committed entries.

Reply:
I think the annotations below are enough so I remove this line.